main-command/src/main/scala/sbt/internal/client/NetworkClient.scala

```diff
@@ -1214,16 +1217,18 @@ object NetworkClient {
   }
 
   def run(configuration: xsbti.AppConfiguration, arguments: List[String]): Int =
```

Comment:
`configuration` is not used

Reply:
Good catch. It is supposed to be used: https://github.com/sbt/sbt/compare/dd2ed16389e1a2c4d082dc6d627b86c8553a432b..f96eeb9b09a9a94b1b5c079fc52497053dd592ae